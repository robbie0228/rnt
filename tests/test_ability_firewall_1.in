board
ability 1 6 7
board
move a down
board
ability 1 2 0
move H up
board
move a down
board
quit
