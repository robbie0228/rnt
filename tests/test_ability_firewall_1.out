Player 1:
Downloaded: 0D, 0V
Abilities: 5
a: D1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
abcSSfgh
...de...
........
........
........
........
...DE...
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: D1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
abcSSfgh
...de...
........
........
........
........
...DE..m
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: ?
========
.bcSSfgh
a..de...
........
........
........
........
...DE..m
ABCSSFGH
========
Player 2:
Downloaded: 0D, 0V
Abilities: 5
A: D1   B: D2   C: D3   D: D4
E: V1   F: V2   G: V3   H: V4

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: D1   b: D2   c: D3   d: D4
e: V1   f: V2   g: V3   h: V4
========
.bcSSfgh
a..de...
w.......
........
........
........
...DE..m
ABCSSFG.
========
Player 2:
Downloaded: 0D, 1V
Abilities: 4
A: ?    B: ?    C: ?    D: ?
E: ?    F: ?    G: ?    H: ?

Player 1:
Downloaded: 0D, 0V
Abilities: 4
a: ?    b: ?    c: ?    d: ?
e: ?    f: ?    g: ?    h: V4
========
.bcSSfgh
...de...
a.......
........
........
........
...DE..m
ABCSSFG.
========
Player 2:
Downloaded: 0D, 1V
Abilities: 4
A: D1   B: D2   C: D3   D: D4
E: V1   F: V2   G: V3   H: V4

